### ISSUE TITLE
Install bug, missing grant in source/create_utplsql_owner.sql

### ISSUE THREAD

**Issue by dmcghan** [utPLSQL/utPLSQL#1150]
I think the `create_utplsql_owner.sql` script also needs to grant `create any context` to the utPLSQL owner for the subsequent `source/install.sql` script to succeed.

https://github.com/utPLSQL/utPLSQL/blob/75037c8ec46aa280058e047015c9a50a60ec6b0c/source/create_utplsql_owner.sql#L32

**Comment by pesse** [comment]
Thanks for reporting, Dan. The issue is also mentioned here: #1147 

IMHO, we should discuss if we provide 2 different paths for installation here:
- One that requires `create any context` during installation
- One that requires the context used by utPLSQL to exist, so people can ask their DBA to create the context instead

From my experience, getting a grant that contains `any` is nearly impossible in a lot of situations

Thoughts @jgebal @PhilippSalvisberg ?

**Comment by PhilippSalvisberg** [comment]
>I think the create_utplsql_owner.sql script also needs to grant create any context to the utPLSQL owner for the subsequent source/install.sql script to succeed.

The context `<username>_info` is created in the `install.sql` script. This script is designed to be executed as a DBA  see ([docs](http://utplsql.org/utPLSQL/latest/userguide/install.html#installing-utplsql)). So this user must have the `create any context` privilege. Therefore I think the `create_utplsql_owner.sql` should not be changed. 

>IMHO, we should discuss if we provide 2 different paths for installation here (...)

I would not introduce two installations paths. It complicates the installation process and I currently do not understand why something like that is required. Let me explain that. 

utPLSQL is either installed in an environment where a developer has access to the DBA user (e.g. personal development environment) or it's installed in environment where a developer does not have access to a DBA user  (e.g. shared development environment, integration test environment, ...). In the former case you can easily use the DBA user for the installation process. And in the latter case you *must* use the DBA user. Yes, you have to ask another person to do the installation. But you have to do that anyway to create the user. So I do not see how an additional installation path leaving the utPLSQL user with a `create any context` privilege would simplify things. 

Maybe @dmcghan can elaborate why using a DBA user is a no-go in his case.

**Comment by pesse** [comment]
Thanks Philipp for clarifying.
I understand the issue now and agree with you. (Have to admit I didn't read the latest docs carefully)

**Comment by dmcghan** [comment]
> Maybe @dmcghan can elaborate why using a DBA user is a no-go in his case.

It's not a "no go"... I try not to use SYSDBA if it can be avoided as I have to get a DBA involved. Something about the way the doc was worded made me think I could do it without SYSDBA, though there would be some tradeoffs. 

For example, this part here:
> The scripts need to be executed by SYSDBA, in order to grant access to DBMS_LOCK and DBMS_CRYPTO system packages.

Is followed by this: 
> Grant on DBMS_LOCK is required only for installation on Oracle versions below 18c. For versions 18c and above, utPLSQL uses DBMS_SESSION.SLEEP so access to DBMS_LOCK package is no longer needed.

(on a related note, perhaps moving to `[standard_hash](https://docs.oracle.com/database/121/SQLRF/functions183.htm#SQLRF55647)` would allow you to remove the dependency on DBMS_CRYPTO for 12+)

While I was able to get things working for the most part, I really wanted the grants to PUBLIC so I ended up going through a DBA. 

Now that I see the intent was always to have the install done by a SYSDBA, I'll go ahead and close this. Thanks!
